docs/devops.md

```diff
@@ -23,7 +23,7 @@ It focuses on unifying Development & IT Operations through the principles of flo
 - **Deployment**: The deployment (or transport) of code to a higher-level environment, up to & including production.
 - **Release**: The defined scope and business objective for a set of features to be delivered in a certain timeline.
 
-## Why Implement DevOps?
+## Why implement DevOps?
```

Comment:
title case?

Reply:
Same here, @xavier506 then remove the question mark, right?